datasets/submit.sh

```diff
@@ -69,9 +68,17 @@ echo \$SLURM_JOB_ID
 source ~/.bashrc
 mamba activate yammbs-dataset-submission
 
+echo "=== CONDA ENV ==="
+mamba list
+echo
+
 echo \$OE_LICENSE
```

Comment:
Not a priority comment or anything, but any reason to echo this? Also, I'm mildly confused that I thought the backslash would escape the $ and it would literally echo "$OE_LICENSE", but the output file seems to have the path of your license.

Reply:
I wonder if you could echo the environment/provenance info to a different file? It might make it harder to parse any actual errors generated during the run

Reply:
I just thought it would be nice to label that section of the output, and the extra echo leaves a paragraph break for jumping around with `}` in vim. I slightly prefer keeping it in a single file just to avoid having many separate files, but it doesn't really matter.

Everything between the `INP`s is a [heredoc](https://en.wikipedia.org/wiki/Here_document) (basically a multiline string literal), so the backslash is to escape the first round of shell expansion at the creation of the heredoc. That `$OE_LICENSE` and the other escaped variables are expanded when Slurm runs the script, not when the user runs `submit.sh`. In this case it probably doesn't matter, if you have `OE_LICENSE` defined in your submitting environment, but `SLURM_JOB_ID`, for example, will be empty without the backslash.

Reply:
SGTM, I don't have a strong preference, I think it's fine this way

Reply:
Ah, I see, thanks for explaining $OE_LICENSE!